libs/staging-provider/tests/OutputProviderInitializerTest.php

```diff
@@ -328,4 +328,43 @@ public function testInitializeOutputTeradata()
         $this->expectException(InvalidOutputException::class);
         $stagingFactory->getTableOutputStrategy(OutputStrategyFactory::WORKSPACE_SNOWFLAKE);
     }
+
+    public function testInitializeOutputBigQuery()
+    {
+        $clientWrapper = new ClientWrapper(
+            new ClientOptions((string) getenv('STORAGE_API_URL'), (string) getenv('STORAGE_API_TOKEN'))
+        );
+        $stagingFactory = new OutputStrategyFactory(
+            $clientWrapper,
+            new NullLogger(),
+            'json'
+        );
+
+        $providerFactory = new ComponentWorkspaceProviderFactory(
+            new Components($clientWrapper->getBasicClient()),
+            new Workspaces($clientWrapper->getBasicClient()),
+            'my-test-component',
+            'my-test-config',
+            new WorkspaceBackendConfig(null)
+        );
+        $init = new OutputProviderInitializer($stagingFactory, $providerFactory, '/tmp/random/data');
+
+        $init->initializeProviders(
+            OutputStrategyFactory::WORKSPACE_BIGQUERY,
+            [
+                'owner' => [
+                    'hasBigquery' => true,
+                    'fileStorageProvider' => 'aws',
+                ],
```

Comment:
Premyslel jsem, proc u testu na ostatni backendy je tam pozapinanych vic backendu. Obvious answer se nabizi, protoze byl nekdo liny a jen pridaval :slightly_smiling_face: Ale nemuze to byt i jako test toho, ze se jejich konfigurace enjak nepomichaji/neprepisou? Pak by mi davalo smysl i tadu nechat vic backendu povolenych.

Reply:
Ja myslim ze byl nekdo linej (i ja). Taky jsem na to koukal:)